.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Welcome to the OpenTracing Go repo! 👋🎉
+
+- Please be respectful and considerate of others when commenting on issues.
+- Please search for existing issues in order to ensure we don't have duplicate bugs/feature requests.
+- Please don't ask questions here. If you have any question head to our gitter chat https://gitter.im/opentracing/public
+- Please provide as much information as possible so we all understand the issue.
+-->
+
+
+## Background
```

Comment:
I tried to fill this once and was fairly confused between Background and Problem.

I would suggest rephrasing, e.g.

## Use Case

Please explain your user story, what you are trying to do, which problem you are trying to solve.

## Problem

What prevents you from solving your use case.